scripts/hermit_test.py

```diff
@@ -1,86 +1,150 @@
 #!/usr/bin/env python
-import os
+import argparse
+import re
 import sys
-from optparse import OptionParser
+from pathlib import Path
+
+
+OWL_NOTHING = 'http://www.w3.org/2002/07/owl#Nothing'
+BOTTOM_OBJECT_PROPERTY = 'http://www.w3.org/2002/07/owl#bottomObjectProperty'
+
+IRI = '<(.*?)>'
+
+EQUIVALENT_CLASSES = r'^EquivalentClasses\( (.*) \)$'
+OBJECT_INVERSE_OF = r'ObjectInverseOf\( ' + IRI + r' \)'
+EQUIVALENT_OBJECT_PROPERTIES = r'^EquivalentObjectProperties\( (.*) \)$'
+
+
+def equivalent_to_nothing(subsumption):
```

Comment:
one instance reported is *http://www.w3.org/2002/07/owl#Nothing*. Which is totally fine (Nothing equivalent to Nothing). Maybe we need to have a special case for it here to avoid reporting it as error?

Reply:
I tried it on an empty ontology: there should be no problems as hermit is not reporting "Nothing equivalent to Nothing".
```
# HermiT results


## Classes equivalent to http://www.w3.org/2002/07/owl#Nothing

No issues detected.


## Subclasses of http://www.w3.org/2002/07/owl#Nothing

No issues detected.


## Object properties equivalent to http://www.w3.org/2002/07/owl#bottomObjectProperty

No issues detected.


## Subproperties of http://www.w3.org/2002/07/owl#bottomObjectProperty

No issues detected.


## Unhandled subsumptions

No issues detected.
```